### PR TITLE
networkctl: Add support to display geneve tunnel info

### PR DESCRIPTION
**Pull request by ssahani**

### CODE REVIEW

File: src/network/networkctl.c [hunk 1]
```diff
@@ -1477,6 +1495,55 @@ static int link_status_one(
...
+                                           TABLE_STRING, "TOS:",
+                                           TABLE_UINT16, info->tos);
+                        if (r < 0)
+                                return table_log_add_error(r);
+                }
+
+                r = table_add_many(table,
+                                   TABLE_EMPTY,
+                                   TABLE_STRING, "Port:",
+                                   TABLE_UINT16, info->tunnel_port);
```
Comment (yuwata) [inline src/network/networkctl.c:1572]: Is the port always set?

File: src/network/networkctl.c [hunk 2]
```diff
@@ -1477,6 +1495,55 @@ static int link_status_one(
...
+                                           TABLE_UINT8, info->ttl);
+                        if (r < 0)
+                                return table_log_add_error(r);
+                }
+
+                if (info->tos > 0) {
+                        r = table_add_many(table,
+                                           TABLE_EMPTY,
+                                           TABLE_STRING, "TOS:",
+                                           TABLE_UINT16, info->tos);
```
Comment (yuwata) [inline src/network/networkctl.c]: UINT8

File: src/network/networkctl.c (hunk 1 above)
Comment (ssahani) [reply src/network/networkctl.c:1572]: yes 
``` networkctl status geneve0                                                                                                                                                          ─╯
● 12: geneve0                                                   
             Link File: /usr/lib/systemd/network/99-default.link
          Network File: n/a                                     
                  Type: geneve                                  
                 State: off (unmanaged)                         
                Driver: geneve                                  
            HW Address: 0e:5a:76:5d:8e:e3                       
                   MTU: 1450 (min: 68, max: 65485)              
                 QDisc: noop                                    
                   VNI: 21                                      
                Remote: 192.168.11.12                           
                  Port: 49431                                   
  Queue Length (Tx/Rx): 1/1    ```

**Comment by ssahani** [comment]
Updated thanks for the review @yuwata . PTAL

File: src/network/networkctl.c (hunk 1 above)
Comment (yuwata) [reply src/network/networkctl.c:1572]: OK. Thanks.

**Comment by yuwata** [comment]
LGTM.

**Comment by yuwata** [comment]
Still LGTM.

**Pull request merged**